packages/fxa-settings/src/lib/auth-errors/auth-errors.test.ts

```diff
@@ -0,0 +1,80 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { composeAuthUiErrorTranslationID, AuthUiErrorNos } from './auth-errors';
+import * as Sentry from '@sentry/browser';
+
+const notAnExistingErrorNumber = 100000;
+const errorWithNoErrorNumber = {
+  message: "I'm an error with no error number!",
+};
```

Comment:
For slightly better TS, you can write `} as AuthUiError;` on L11, as well as on L15.